medicines/api/src/document.rs

```diff
@@ -37,6 +37,45 @@ impl From<IndexResult> for Document {
 
 pagination! {Documents, DocumentEdge, Document}
 
+fn get_document_edges(docs: Vec<Document>, offset: i32) -> Vec<DocumentEdge> {
```

Comment:
Love this approach.

Would `build_` be a better prefix than `get_`? To me, "get" implies retrieving from some data source, whereas "build" would be about building structures.